### PR TITLE
Services onboarding

### PR DESCRIPTION
**Pull request by vsapronov**
Related to this: https://github.com/ModaOperandi/agora/issues/53

### CODE REVIEW

File: recommendations/services-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+## Recommended Levels
+
+| Stack  | Currently Supported | Recommended Level            |
+| :----- | :------------------ | :--------------------------- |
+| scala  | yes                 | Build + Deployment + Codegen |
+| python | no                  | Build + Deployment           |
+| java   | no                  | Deployment                   |
+| ruby   | no                  | Deployment                   |
+| flow   | no                  | ???                          |
```
Comment (rlmartin) [inline recommendations/services-onboarding.md:43]: As discussed, can you add JavaScript here? And maybe put `flow` under `Deployment` or completely remove it if it's not a microservice.

**Comment by rlmartin** [comment]
I think a nice result from the different levels of support is that we can color-code both stacks and services with red/yellow/green, i.e. call out which level of support for each stack plus whether or not it is fully implemented. This would trickle down to the services within that stack as well, based on the color of the stack and that service's migration to being fully on the platform for that stack. We could then have a sheet of all services with color-coding and would get a good sense of the platform coverage/health of the entire fleet.

File: recommendations/services-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+## Required Work
+
+1. Python stack should be supported for deployment.
+2. Java stack should be supported for deployment.
+3. Standard build job for Python AIOHTTP service should be developed to build Python services.
+4. Some services have AWS Lambda functions added. Lambda functions are not currently supported on platform. There are some plans to potentially provide such support.
+5. RDS storage support is needed for few services. Yet to investigate if the storage is Mojo MySQL or separate database.
+
+Mojo-related issues:
+1. MySQL database access is not provided from platform. Each individual case should be considered separately.
```
Comment (jackfarzan) [inline recommendations/services-onboarding.md:54]: this is no longer an issue

**Comment by rlmartin** [comment]
Here is a list of microservice platform support, to be filled in and evaluated for overall platform coverage/risk/prioritization: https://docs.google.com/spreadsheets/d/1TczlY048X3RktS9AK_QHpLpMKGqLVCQGcchpyZXQXEw/edit?usp=sharing

**Pull request closed without merging**